python/python/tests/test_async_client.py

```diff
@@ -3271,6 +3271,22 @@ async def test_time(self, redis_client: TRedisClient):
         assert int(result[0]) > current_time
         assert 0 < int(result[1]) < 1000000
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_lastsave(self, redis_client: TRedisClient):
+        yesterday = date.today() - timedelta(1)
+        yesterday_unix_time = time.mktime(yesterday.timetuple())
+
+        result = await redis_client.lastsave()
+        assert result > yesterday_unix_time
+
+        if isinstance(redis_client, RedisClusterClient):
+            result = await redis_client.lastsave(AllNodes())
```

Comment:
From my recent experience we need to always test with single node and multi node routes.
Maybe you need to move everything in `if` to another test method.

Reply:
I think its fine to have the cluster part in the same test like I'm doing here since that pattern exists elsewhere in this file. I'll add a test with a single node route though